.github/workflows/continuous-integration.yml

```diff
@@ -51,8 +51,11 @@ jobs:
         uses: conda-incubator/setup-miniconda@v2
         with:
           auto-update-conda: true
+          environment-file: environment.yml
           python-version: ${{ matrix.python-version }}
-      - name: Update Conda's environemnt
-        run: conda env update -f environment.yml -n test
+      - name: update certs so PROJ can download the us_noaa_cshpgn.tif file
+        run: |
+          sudo mkdir -p /etc/pki/tls/certs
+          sudo curl https://curl.se/ca/cacert.pem -o /etc/pki/tls/certs/ca-bundle.crt
```

Comment:
This is wild, I can't believe it hasn't come up before. But so it goes, good catch.

Reply:
I think likely it's specific to NAD83 / UTM zones that NAIP use -- I don't know of any of the other packages that use that projection. I think the test also never test if the reprojected geometry is correct, so the (Inf, Inf) point was never asserted, and it didn't actually cause a test failure til I tried to use the centroid.

Reply:
I was more referring to the "need to download certs" issue.

Reply:
that's what I mean -- I think NAD83 / UTM zones are the only ones that require PROJ to actually download something over HTTPS, so none of the other stactools packages need the certs.

Reply:
Gotcha -- I guess maybe I'm surprised Github's runners aren't set up to download SSL out-of-the-box? Anyways, all good.